Profile renaming fails

I'm trying to rename desktop profile in FleetCommander WebUI.
Operation fails with:

![Screenshot_20190312_211221](https://user-images.githubusercontent.com/31205609/54225023-d7dc3300-450b-11e9-8c84-3c233fb207c3.png)


```
org.freedesktop.FleetCommander[12584]: FC: [DEBUG] FreeIPAConnector: Updating profile test and renaming to test_
org.freedesktop.FleetCommander[12584]: FC: [DEBUG] FreeIPAConnector: Updating profile test_ with data: {'cn': 'test', 'name': 'test_', 'description': '', 'priority': 50, 'settings': {}, 'groups': [], 'users': [], 'hosts': [], 'hostgroups': [], 'oldname': 'test'}
org.freedesktop.FleetCommander[12584]: FC: [ERROR] FreeIPAConnector: Error updating profile test_: Unknown option: rename
org.freedesktop.FleetCommander[12584]: FC: [ERROR] Error saving profile test: (test_) Unknown option: rename
```

--- a/plugin/ipaserver/plugins/deskprofile.py
+++ b/plugin/ipaserver/plugins/deskprofile.py
@@ -141,8 +141,7 @@ class deskprofile(LDAPObject):
         'cn', 'description',
     ]
     uuid_attribute = 'ipauniqueid'
-    rdn_is_primary_key = True
-
+    allow_rename = True
     managed_permissions = {
         'System: Read FleetCommander Desktop Profile': {
             'ipapermbindruletype': 'all',
@@ -263,7 +262,7 @@ class deskprofilerule(LDAPObject):
         'seealso',
     ]
     uuid_attribute = 'ipauniqueid'
-    rdn_is_primary_key = True
+    allow_rename = True
     attribute_members = {
         'memberuser': ['user', 'group'],
         'memberhost': ['host', 'hostgroup'],

